### DISCUSSION TITLE
[Examples] Provide auth example for "AWS Amplify with NextJS" sample

### DISCUSSION THREAD

**Discussion by thesobercoder** [vercel/next.js#62355] [Help] [answered]
# Feature request

## Is your feature request related to a problem? Please describe.

The current sample provided for AWS Amplify doesn't have code to show how to use AWS Amplify auth. Also not clear is how to do auth like shown in the `Auth0` with APIs.

## Describe the solution you'd like

A more robust sample with API and auth will give newcomers more confidence in adopting the sample for more complex applications.

**Comment by Rajat379** [comment]
Hello, I would like to work on this feature.

**Comment by balazsorban44** [comment]
You are welcome to! :green_heart:

**Comment by utkarsh4321** [comment]
Hi, @balazsorban44 what do you really want can you explain it more?

**Comment by jfisher446** [comment, marked as answer]
The requested example is part of a recent write-up on the AWS blog:

- [Deploy a Next.js 13 app with authentication to AWS Amplify](https://aws.amazon.com/blogs/mobile/deploy-a-next-js-13-app-with-authentication-to-aws-amplify/)
